src/components/templates/Search/utils.ts

```diff
@@ -100,12 +100,21 @@ export function getSearchQuery(
   searchTerm = addTypeFilterToQuery(searchTerm, serviceType)
   searchTerm = addPriceFilterToQuery(searchTerm, priceType)
 
+  // HACK: resolves the case sensitivity related to dataTokenInfo.symbol
+  searchTerm = searchTerm.toUpperCase()
+
   return {
     page: Number(page) || 1,
     offset: Number(offset) || 21,
     query: {
       query_string: {
-        query: `${searchTerm} -isInPurgatory:true`
+        query: `*${searchTerm}* -isInPurgatory:true`,
+        fields: [
+          'dataTokenInfo.name',
+          'dataTokenInfo.symbol',
+          'service.attributes.main.name'
+        ],
+        default_operator: 'AND'
```

Comment:
wasn't this an `OR` operator before? And worth checking for other fields like owner or description which was working before and should be added to `fields` array

Reply:
Before we didn't have a specified operator, so by default `OR` was used indeed. In our search, I believe we need a match with all terms specified in search field, that's why I choose `AND`, as they explain in `default_operator` part : https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html. 

Sure, we can add the author and description as well.

Reply:
I see your point, but we need a solution which cooperates with strings introduced in the search field as token symbol, asset name, author,  description. There's no specific  separation between them. Thus, for a search token we won't search for token symbol specifically.